Content.Server/Chat/ChatManager.cs

```diff
@@ -224,6 +224,27 @@ public void SendDeadChat(IPlayerSession player, string message)
             _netManager.ServerSendToMany(msg, clients.ToList());
         }
 
+        public void SendAdminDeadChat(IPlayerSession player, string message)
+        {
+            // Check if message exceeds the character limit
+            if (message.Length > MaxMessageLength)
+            {
+                DispatchServerMessage(player, Loc.GetString(MaxLengthExceededMessage, MaxMessageLength));
+                return;
+            }
+
+            var clients = _playerManager
+                .GetPlayersBy(x => x.AttachedEntity != null && x.AttachedEntity.HasComponent<GhostComponent>())
+                .Select(p => p.ConnectedClient);
```

Comment:
This should definitely not be duplicated from the other method. Make a helper for it or something.